nbs/99_manuscript/time_test/07-time_test-plot.ipynb

```diff
@@ -0,0 +1,1643 @@
+{
```

Comment:
<p>Could you use a paired color palette instead, so both CCC (1 core) and CCC (3 cores) were e.g. shades of blue, and MIC (1 core) and MIC (3 cores) were shades of green, etc?</p><p> </p><p>Examples from R: https://bookdown.org/hneth/ds4psy/D-2-apx-colors-essentials.html#hybrid-color-palettes</p><p> </p><p>I think it might make finding patterns in this graph a bit easier</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/clustermatch-gene-expr/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>Yes, fantastic! It's much better now. Although this is another internal figure (the final one is much simplified), this really improved the visuals.</p>